src/Process/Deploy/CheckEnvVarMageErrorReportDirNestingLevel.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\MagentoCloud\Process\Deploy;
```

Comment:
why it is not in Magento\MagentoCloud\Config\Validator\Deploy namespace?

Reply:
Changed